rllib/models/catalog.py

```diff
@@ -107,19 +108,22 @@ class ModelCatalog:
     @staticmethod
     @DeveloperAPI
     def get_action_dist(action_space,
-                        config,
+                        config=None,
                         dist_type=None,
                         torch=None,
-                        framework="tf"):
+                        framework="tf",
+                        **kwargs):
         """Returns a distribution class and size for the given action space.
 
         Args:
             action_space (Space): Action space of the target gym env.
-            config (dict): Optional model config.
+            config (Optional[dict]): Optional model config.
             dist_type (Optional[str]): Identifier of the action distribution.
-            torch (bool): Obsoleted: Whether to return PyTorch Model and
+            torch (bool): Deprecated: Whether to return PyTorch Model and
```

Comment:
You can just make this change directly, since it's DeveloperAPI. No deprecation needed.